### PR TITLE
Server shinan

### PR DESCRIPTION
**Pull request by ShinanSun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,186 @@
...
+- User authentication
+- Display the book of the month
+- Explore trending books from the store
+- View user libary and purchased book
+- Explore user libary and
+- Connect and add friends
+- Create bookclub and bookclub event
+- Email notification for bookclub event
+
+### Built With
```
Comment (mattBman23) [inline README.md:80]: Can we use logos for our tech stack like
image

File: README.md [hunk 2]
```diff
@@ -1 +1,186 @@
...
+1. Get a free API Key at [https://example.com](https://example.com)
+2. Clone the repo
+   ```sh
+   git clone https://github.com/hratx-blue-ocean/Papercuts.git
+   ```
+3. Install NPM packages
+   ```sh
+   npm install
+   ```
+4. Enter your API in `config.js`
```
Comment (mattBman23) [inline README.md:118]: Need to tell them where. Is it in the client folder? If so where in the client

**Review by mattBman23 — CHANGES_REQUESTED** [review]

**Review by jfleming9357 — CHANGES_REQUESTED** [review]
Just needs a fix on firebase.js

File: README.md (hunk 2 above)
Comment (ShinanSun) [reply README.md:118]: this is just a template to start with matt, You guys need to help me writing other parts

File: Docs/firebase-config/firebase.js [hunk 3]
```diff
@@ -1,5 +1,19 @@
 import Firebase from 'firebase/app';
 import 'firebase/storage';
+<<<<<<< HEAD
+
```
[the remark this answers is not in the document]
Comment (ShinanSun) [reply Docs/firebase-config/firebase.js]: Sorry, I thought I git pushed it. my bad

File: README.md (hunk 1 above)
Comment (ShinanSun) [reply README.md:80]: Ok, I am adding it to there

**Review by jfleming9357 — APPROVED** [review]

**Review by daniel-he-dev — APPROVED** [review]

**Pull request merged**